index.html

```diff
@@ -1,117 +1,154 @@
-      }
-      gtag('js', new Date())
-
-      gtag('config', 'G-NM34VFTB3N')
-    </script>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="wclassth=device-wclassth, initial-scale=1.0" />
-    <link rel="stylesheet" href="./assets/css/style.css" />
-    <title>Chimoney Community Website</title>
-
-    <!-- Favicons -->
-    <link href="assets/images/Chimoney.png" rel="icon" />
-  </head>
-
-  <body>
-    <!-- Navbar -->
-    <!-- End of Navbar -->
-
-    <!-- Hero section -->
-    
-    <!--featured post-->
-    <div class="feature-container">
-        <div class="center-container">
-            <p class="feature-heading">Featured posts</p>
-            <div class="line">
-              <hr class="line1">
-              <hr class="line2">
-            </div>   
-            <div class="data" id="data"></div>
-            
-            <button class="see-more">see more</button>
-        </div>
+
+<head>
+  <!-- Global site tag (gtag.js) - Google Analytics -->
+  <script async src="https://www.googletagmanager.com/gtag/js?class=G-NM34VFTB3N"></script>
+  <script>
+    window.dataLayer = window.dataLayer || []
+
+    function gtag() {
+      dataLayer.push(arguments)
+    }
+    gtag('js', new Date())
+
+    gtag('config', 'G-NM34VFTB3N')
+  </script>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="wclassth=device-wclassth, initial-scale=1.0" />
+  <link rel="stylesheet" href="./assets/css/style.css" />
+  <title>Chimoney Community Website</title>
+
+  <!-- Favicons -->
+  <link href="assets/images/Chimoney.png" rel="icon" />
+  <!-- Poppins Google font link-->
+  <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+
+</head>
+
+<body>
+  <!-- Navbar -->
+  <div class="navbar">
+    <nav>
+      <div class="navbar-logo">
+        <img src="assets/images/navlogo.svg" alt="Chimoney">
+      </div>
+      <div class="nav-link">
+        <ul>
+          <li><a href="#">Product</a></li>
+          <li><a href="#">Blog</a></li>
+          <li><a href="#">Community</a></li>
+          <li><a href="#">About</a></li>
+        </ul>
+      </div>
+      <div class="nav-item">
+        <button id="sign-in">Sign in</button>
+        <button id="contact-sales">Contact sales</button>
+      </div>
+    </nav>
+  </div>
+  <!-- End of Navbar -->
+
+  <!-- Hero section -->
+  <div class="hero-section">
+    <div class="heading">
+      <h1>Find your place in the</h1>
+      <h1>Chimoney Community</h1>
+      <h3>Whether you're a beginner, expert, developer, or</h3>
+      <h3>designer, everyone is welcome to join our</h3>
+      <h3>international network of users.</h3>
+      <button id="contact-sales">JOIN OUR COMMUNITY</button>
+    </div>
+    <div class="header-image">
+      <img src="assets/images/Illustrations.svg" alt="Image">
     </div>
+  </div>
+  <!--End of Hero section-->
 
```

Comment:
Hi @Sarthak0006 please submit changes for only NavBar and Hero section. 
The other changes interfere with the other sections.